client/syntaxes/coq.tmLanguage

```diff
@@ -45,7 +45,7 @@
 
         <dict>
             <key>match</key>
-            <string>(Lemma|Theorem|Goal|Example)\s+([a-zA-Z][a-zA-Z0-9_']*)</string>
+            <string>(Lemma|Theorem|Goal|Example)\s+(([\p{L}-]|[0-9]|[_'])+)</string>
```

Comment:
Doesn't this allow identifiers beginning with numbers?

Reply:
Good catch; will fix shortly.